docs/reference/commandline/build.md

```diff
@@ -345,6 +345,13 @@ This will use a file called `Dockerfile.debug` for the build instructions
 instead of `Dockerfile`.
 
 ```bash
+$ docker build -f - . < ../Dockerfile
```

Comment:
Wouldn't that fail because the Dockerfile is outside of the context? `cat` isn't any different.

I could do something like `./render.sh | docker build -f - .` , but it might not be clear what render.sh is doing.